ansible/roles/ubuntu-worker/tasks/rebar.yml

```diff
@@ -0,0 +1,37 @@
+- name: create rebar group
+  group:
+    name: rebar
+    state: present
+
+- name: create rebar user
+  user:
+    name: rebar
+    createhome: yes
+    group: rebar
+    state: present
+
+- name: download rebar
+  get_url:
+    url: https://github.com/rebar/rebar/archive/{{ rebar_version }}.tar.gz
```

Comment:
No need in rebar: ./configure script [takes care](https://github.com/apache/couchdb/blob/master/configure#L652-L667) about it. While you stay with `make *` commands, it will be used and right version.